config/config.go

```diff
@@ -128,6 +130,29 @@ func (cfg *AuctionTimeouts) validate(errs configErrors) configErrors {
 	return errs
 }
 
+func (data *ExternalCache) validate(errs configErrors) configErrors {
+	if !(data.Host == "" && data.Path == "") {
```

Comment:
It's better to avoid multiple levels of indentation if it can be avoided. IMHO, I think this looks cleaner:
```
if (data.Host == "" && data.Path == "") {
 return nil
}
```